### PR TITLE
[ADD] developer: add js http service

### PR DESCRIPTION
**Pull request by SimonGenin**

### CODE REVIEW

**Comment by SimonGenin** [comment]
@ged-odoo @odoo/doc-review

**Comment by ged-odoo** [comment]
it's better to ping doc review only after the content has been reviewed by js team @SimonGenin

File: content/developer/reference/frontend/services.rst [hunk 1]
```diff
@@ -341,6 +343,40 @@ Here, it is called in webclient.js to make it visible everywhere for the example
...
+Http Service
+------------
+
+Overview
+~~~~~~~~
+
+* Technical name: `http`
+* Dependencies: None
+
+While most interaction between the client and the server in odoo are `RPCs`, lower level
```
Comment (ged-odoo) [inline content/developer/reference/frontend/services.rst:357]: `interaction` -> `interactions`

Comment (ged-odoo) [reply content/developer/reference/frontend/services.rst:357]: you can add `(XMLHTTPRequest)` after `RPCs`

File: content/developer/reference/frontend/services.rst [hunk 2]
```diff
@@ -341,6 +343,40 @@ Here, it is called in webclient.js to make it visible everywhere for the example
...
+control on requests may sometimes be required.
+
+This service provides a way to send `get` and `post` http requests.
+
+API
+~~~
+
+.. js:function:: async get(route [,readMethod = "json"])
```
Comment (ged-odoo) [inline content/developer/reference/frontend/services.rst:365]: `get(route[,readMethod = "json"])` (no space after route)

File: content/developer/reference/frontend/services.rst [hunk 3]
```diff
@@ -341,6 +343,40 @@ Here, it is called in webclient.js to make it visible everywhere for the example
...
+Overview
+~~~~~~~~
+
+* Technical name: `http`
+* Dependencies: None
+
+While most interaction between the client and the server in odoo are `RPCs`, lower level
+control on requests may sometimes be required.
+
+This service provides a way to send `get` and `post` http requests.
```
Comment (ged-odoo) [inline content/developer/reference/frontend/services.rst:360]: maybe link this: https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods

File: content/developer/reference/frontend/services.rst [hunk 4]
```diff
@@ -341,6 +343,40 @@ Here, it is called in webclient.js to make it visible everywhere for the example
...
+
```
Comment (ged-odoo) [inline content/developer/reference/frontend/services.rst:361]: one example would be cool

**Review by ged-odoo — APPROVED** [review]

**Comment by AntoineVDV** [comment]
> it's better to ping doc review only after the content has been reviewed by js team @SimonGenin

I'd even prefer if you ping us through the "request review" block on the top-right corner so that I can dismiss unintended reviews and manage which PR must still be reviewed.

File: content/developer/reference/frontend/services.rst (hunk 2 above)
Comment (AntoineVDV) [inline content/developer/reference/frontend/services.rst:365]: You'll probably want to fine-tune a bit the way the square brackets look with the new style of js docstrings.

**Review by AntoineVDV — COMMENTED** [review]
@robodoo delegate+

**Comment by SimonGenin** [comment]
@robodoo r+

**Pull request closed without merging**